__tests__/Organization.js

```diff
@@ -249,6 +249,65 @@ describe('Organizations basic', () => {
 
     patchOrganizationRequest.done();
   });
+
+  it('allows to remove a member from an organization', async () => {
+    getMockCall(`/v4/organizations/${ORGANIZATION}/credentials/`);
+    getMockCall('/v4/organizations/', orgsResponse);
+    getMockCall(`/v4/organizations/${ORGANIZATION}/`, orgResponse);
+    const patchOrganizationRequest = nock(API_ENDPOINT)
+      .intercept(`/v4/organizations/${orgResponse.id}/`, 'PATCH')
+      .reply(StatusCodes.Ok, {
+        code: 'RESOURCE_UPDATED',
+        message: `The organization with ID ${orgResponse.id} has been updated.`,
+      });
+
+    const { getByText, findByText, findByTestId } = renderRouteWithStore(
+      `${BASE_ROUTE}/${orgResponse.id}`
+    );
+
+    // users
+    const usersTable = await findByTestId('org-detail-users-wrapper');
+    expect(usersTable.querySelector('tbody > tr > td').textContent).toBe(
+      orgResponse.members[0].email
+    );
+
+    const removeUserTableButton = usersTable.querySelector(
+      'tbody > tr > td:nth-of-type(3) > button'
```

Comment:
Why not use a test id, as the structure might change in the future?

Reply:
+1

Reply:
The tables on the organization detail component uses an external component. I'll look into adding test-ids

Reply:
removed querySelector from this and the organization detail view test